onediff_comfy_nodes/extras_nodes/nodes_oneflow_optimizer.py

```diff
@@ -0,0 +1,718 @@
+                       f'is_onediff_quant_available={is_onediff_quant_available()}')
+
+        return (
+            OnelineQuantizationOptimizerExecutor(
+                conv_percentage=quantized_conv_percentage,
+                linear_percentage=quantized_linear_percentage,
+                conv_compute_density_threshold = conv_compute_density_threshold,
+                linear_compute_density_threshold = linear_compute_density_threshold,
+            ),
+        )
+
+class OneDiffDeepCacheCheckpointLoaderSimple(CheckpointLoaderSimple):
+    @classmethod
+    def INPUT_TYPES(s):
+        return {
+            "required": {
+                "ckpt_name": (folder_paths.get_filename_list("checkpoints"),),
+                "vae_speedup": (["disable", "enable"],),
+                "static_mode": (["enable", "disable"],),
+                "cache_interval": (
+                    "INT",
+                    {
+                        "default": 3,
+                        "min": 1,
+                        "max": 1000,
+                        "step": 1,
+                        "display": "number",
+                    },
+                ),
+                "cache_layer_id": (
+                    "INT",
+                    {"default": 0, "min": 0, "max": 12, "step": 1, "display": "number"},
+                ),
+                "cache_block_id": (
+                    "INT",
+                    {"default": 1, "min": 0, "max": 12, "step": 1, "display": "number"},
+                ),
+                "start_step": (
+                    "INT",
+                    {
+                        "default": 0,
+                        "min": 0,
+                        "max": 1000,
+                        "step": 1,
+                        "display": "number",
+                    },
+                ),
+                "end_step": (
+                    "INT",
+                    {"default": 1000, "min": 0, "max": 1000, "step": 0.1,},
+                ),
+            }
+        }
+
+    CATEGORY = "OneDiff/Loaders"
+    FUNCTION = "onediff_load_checkpoint"
+
+    @torch.no_grad()
+    def onediff_load_checkpoint(
+        self,
+        ckpt_name,
+        vae_speedup,
+        output_vae=True,
+        output_clip=True,
+        static_mode="enable",
+        cache_interval=3,
+        cache_layer_id=0,
+        cache_block_id=1,
+        start_step=0,
+        end_step=1000,
+    ):
+        # CheckpointLoaderSimple.load_checkpoint
+        modelpatcher, clip, vae = self.load_checkpoint(
+            ckpt_name, output_vae, output_clip
+        )
+        op = OptimizerScheduler(DeepcacheOptimizerExecutor(
+               cache_interval=cache_interval,
+                cache_layer_id=cache_layer_id,
+                cache_block_id=cache_block_id,
+                start_step=start_step,
+                end_step=end_step,
+        ))
+
+        modelpatcher = op.compile(modelpatcher, ckpt_name=ckpt_name)
```

Comment:
```suggestion
        modelpatcher = OptimizerScheduler(DeepcacheOptimizerExecutor(
               cache_interval=cache_interval,
                cache_layer_id=cache_layer_id,
                cache_block_id=cache_block_id,
                start_step=start_step,
                end_step=end_step,
        ))(modelpatcher, ckpt_name=ckpt_name)
```

我看 scheduler 里重写了 `__call__` ，但是所有的 scheuler 调用都还是用的 compile，这是为什么呢。

或者说为什么重写 `__call__`，但是不倾向于使用它呢。

Reply:
当时想的是体现 compile， 已修复为统一用 `__call__ `调用